contracts/test-migrated/src/lib.rs

```diff
@@ -59,4 +59,6 @@ pub mod interface {
             )
         }
     }
+
+    impl<Chain: CwEnv> cw_blob::interface::MigrationFromBlob<Chain> for MigratedBlob<Chain> {}
```

Comment:
We should be able to create a blanket implementation for this trait? 

impl<Chain: CwEnv> cw_blob::interface::MigrationFromBlob<Chain> for T<Chain> where T: ContractInstance<Chain> + Uploadable + ... {}

Reply:
> We should be able to create a blanket implementation for this trait?
> 
> impl<Chain: CwEnv> cw_blob::interface::MigrationFromBlob for T where T: ContractInstance + Uploadable + ... {}

Why would we want to do that? Not every contract would be able to migrate from that contract

Reply:
Ahhh that's true. But if it can't what's the point?

Reply:
We'll leave it like this :)

Reply:
> Ahhh that's true. But if it can't what's the point?

We import it inside Abstract, but in our case it would be weird for account to implement it. I like blanket impl when it's applicable on all use-cases